internal/lpastore/lpa.go

```diff
@@ -246,6 +246,7 @@ type Lpa struct {
 	Submitted                                  bool
 	Paid                                       bool
 	IsOrganisationDonor                        bool
+	IsPaperDonor                               bool
```

Comment:
Would it be better to add a `Channel` to `Donor`, then set that? We could then set it to the correct option when a donor is created in dynamo/read from lpastore